packages/common/src/core/schemas/internal/follow/FollowCardUiSchema.ts

```diff
@@ -16,80 +39,100 @@ export const buildUiSchema = (
-        scope: "/properties/callToActionText",
-        type: "Control",
-        options: {
-          type: "textarea",
-          rows: 4,
-        },
-      },
-      {
-        label: "Alignment",
-        scope: "/properties/callToActionAlign",
-        type: "Control",
-        options: {
-          control: "hub-field-input-alignment",
-          layout: "inline-space-between",
-        },
-      },
-      {
-        label: "Follow",
-        scope: "/properties/buttonText",
-        type: "Control",
-      },
-      {
-        label: "Unfollow",
-        scope: "/properties/unfollowButtonText",
-        type: "Control",
-      },
-      {
-        label: "Button Alignment",
-        scope: "/properties/buttonAlign",
-        type: "Control",
-        options: {
-          control: "hub-field-input-alignment",
-          layout: "inline-space-between",
-        },
+        type: "Section",
+        labelKey: "callToAction.title",
+        rule: HIDE_FOR_NO_ENTITY_ID,
+        elements: [
+          {
+            labelKey: "callToAction.description",
+            scope: "/properties/callToActionText",
+            type: "Control",
+            options: {
+              type: "textarea",
+              rows: 4,
+            },
+          },
+          {
+            labelKey: "callToAction.alignment",
+            scope: "/properties/callToActionAlign",
+            type: "Control",
+            options: {
+              control: "hub-field-input-alignment",
+            },
+          },
+        ],
       },
       {
-        label: "Button Style",
-        scope: "/properties/buttonStyle",
-        type: "Control",
+        type: "Section",
+        labelKey: "followButton.title",
         options: {
-          control: "hub-field-input-select",
-          labels: ["Solid Background", "Outline"],
+          labelKey: "followButton.description",
         },
+        rule: HIDE_FOR_NO_ENTITY_ID,
+        elements: [
+          {
+            labelKey: "followButton.buttonText",
+            scope: "/properties/buttonText",
+            type: "Control",
+            options: {
+              control: "hub-field-input-input",
+            },
+          },
+          {
+            labelKey: "followButton.unfollowButtonText",
+            scope: "/properties/unfollowButtonText",
+            type: "Control",
+            options: {
+              control: "hub-field-input-input",
+            },
+          },
+          {
+            labelKey: "followButton.alignment",
+            scope: "/properties/buttonAlign",
+            type: "Control",
+            options: {
+              control: "hub-field-input-alignment",
+            },
+          },
+          {
+            labelKey: "followButton.buttonStyle",
+            scope: "/properties/buttonStyle",
+            type: "Control",
+            options: {
+              control: "hub-field-input-select",
+              labels: ["Solid Background", "Outline"],
```

Comment:
Should these labels be translated? You should be able to use the same `{{...:translate}}` syntax as long as you use `interpolateTranslations` when you load the `uiSchema`.

Reply:
Good catch!